cmd/version/version.go

```diff
@@ -64,3 +68,62 @@ func displayVersion(cmd *cobra.Command) {
 		Detail:  true,
 	})
 }
+
+func GetLatestVersion() (string, error) {
+	client := &http.Client{
+		CheckRedirect: func(req *http.Request, via []*http.Request) error { // no redirect
+			return http.ErrUseLastResponse
+		},
+	}
+	resp, err := client.Get("https://github.com/cisco-open/fsoc/releases/latest")
+	if err != nil {
+		return "", err
+	}
+	split := strings.Split(resp.Header.Get("Location"), "/")
```

Comment:
you may want to give a comment with an example of what the redirect string is expected to look like; will make understanding this easier